keps/sig-node/34-sysctl-fields/README.md

```diff
@@ -188,15 +256,176 @@ Validation checks implemented as part of [#27180](https://github.com/kubernetes/
 We need to assure backward compatibility, i.e. object specifications with `sysctl` annotations
 must still work after the graduation.
 
-## Graduation Criteria
+## Design Details
+
+All of the above details were copied out of earlier proposals. For graduation, the PRR template below is completed.
+
+### Test Plan
+
+- Unit tests and e2es for all applicable changes.
+- Any required conformance tests for graduation.
+
+### Graduation Criteria
+
+#### Alpha
+
+* add sysctl support to pods
+* e2e tests
+
+Alpha since 1.4.
+
+#### Beta
 
 * API changes allowing to configure the pod-scoped `sysctl` via `spec.securityContext` field.
 * API changes allowing to configure the cluster-scoped `sysctl` via `PodSecurityPolicy` object
-* Promote `--experimental-allowed-unsafe-sysctls` kubelet flag to kubelet config api option
 * feature gate enabled by default
-* e2e tests
+
+Beta since 1.11.
+
+#### Graduation
+
+* Promote `--experimental-allowed-unsafe-sysctls` kubelet flag to kubelet config api option
```

Comment:
this flag does not exist.

it was changed to `allowed-unsafe-sysctls` and no further action is needed.

Reply:
This was done, but it was done post-beta so I will keep it in GA criteria and add it into implementation history.

For some reason [the issue](https://github.com/kubernetes/kubernetes/issues/61669) was still open despite this having been fixed so I closed that issue.

Reply:
https://github.com/kubernetes/enhancements/compare/4f5778734ea946d9793463df9bf352d4adb46ad0..61bc65c3422cedba4fb09da71b8f2c3cb605e7ee